readonly/middleware.py

```diff
@@ -17,7 +17,7 @@ def __init__(self, request):
         self['Location'] = iri_to_uri(referer or "/")
 
 
-class DatabaseReadOnlyMiddleware(object):
+class DatabaseReadOnlyMiddleware(django.utils.deprecation.MiddlewareMixin):
```

Comment:
Thanks for this. I need to make an adjustment to it, and then I can release a new version.

Reply:
@streeter I figured more adjustments were necessary. I am just appreciative the the responsiveness. I wanted to help in any way I can. Thanks again for having this repo.

Reply:
What were the adjustments that you found?

Reply:
For now this was the only adjustment I wanted to make as a transition step. [There are docs that highlight how the new middleware behavior works](https://docs.djangoproject.com/en/1.11/topics/http/middleware/#upgrading-middleware). Likely will require a new class or some more adjustments to the current class to be fully backwards and forwards compatible. Not sure at the moment.